client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import * as React from 'react';
+import domainConnectedSuccess from 'calypso/assets/images/illustrations/domain-success.svg';
+import { buildCtaForProfessionalEmail } from 'calypso/my-sites/checkout/checkout-thank-you/domains/thank-you-content/index';
+import { domainMappingSetup } from 'calypso/my-sites/domains/paths';
+import { DomainThankYouParams, DomainThankYouProps } from '../types';
+
+const DomainRegistrationThankYouProps = ( {
+	domain,
+	email,
+	hasProfessionalEmail,
+	selectedSiteSlug,
+}: DomainThankYouParams ): DomainThankYouProps => {
+	const professionalEmail = buildCtaForProfessionalEmail(
+		{
+			domain,
+			email,
+			hasProfessionalEmail,
+			selectedSiteSlug,
+		},
+		'registration',
+		false
+	);
+
+	const returnProps: DomainThankYouProps = {
+		sections: [
+			{
+				sectionKey: 'domain_registration_whats_next',
+				sectionTitle: translate( 'What’s next?' ),
+				nextSteps: [
+					{
+						stepKey: 'domain_registration_whats_next_plugin_setup',
+						stepTitle: translate( 'Organize your domains' ),
+						stepDescription: translate(
+							'Set up a primary domain, connect other domains and make sure people can find your site'
+						),
+						stepCta: (
+							<Button
+								href={ domainMappingSetup( selectedSiteSlug, domain ) }
+								className={ 'domain-registration__thank-you-button domain-thank-you__button' }
+								primary
+								busy={ false }
+								disabled={ false }
+							>
+								{ translate( 'Manage domains' ) }
+							</Button>
+						),
+					},
+					professionalEmail,
+				],
+			},
+		],
+		thankYouImage: { alt: translate( 'Domain Connected' ), src: domainConnectedSuccess },
```

Comment:
The alt text should be `Domain Registered`, and the variable name should also reflect that this is a registration.
```suggestion
		thankYouImage: { alt: translate( 'Domain Registered' ), src: domainConnectedSuccess },
```